app/main.js

```diff
@@ -301,13 +281,7 @@ function createContributorSettingsWindow () {
   }
 }
 
-function createSyncPreferencesWindow () {
-  if (syncPreferencesWindow) {
-    syncPreferencesWindow.show()
-    return
-  }
-
-  const syncPreferencesUrl = 'https://my.stretchly.net/app/v1/sync'
+function newStretchlyWindow (syncPreferencesUrl) {
```

Comment:
I don't think we should do this. `syncPreferencesWindow` and `myStretchlyWindow` might look similar (in behavior and params) now but it needn't need to be in future. 

You are also mixing them together: in `newStretchlyWindow` you use `syncPreferencesWindow` and then do `myStretchlyWindow = newStretchlyWindow(myStretchlyUrl)`. We could make that a param but I don't think it's safe to assume they will stay the same. 
For example 
```
    width: 1000,
    height: 700,
```
will be changing based on how the website looks. We could also parametrize that but then, we end up the same as `BrowserWindow` itself.